ziggurat-core-geoip/src/location.rs

```diff
@@ -0,0 +1,68 @@
+use serde::{Deserialize, Serialize};
+
+/// Geographical location
+#[derive(Copy, Clone, Deserialize, Serialize)]
+pub struct Location {
+    /// Latitude value
+    pub latitude: f64,
+    /// Longitude value
+    pub longitude: f64,
+}
+
+impl Location {
+    /// Create a new Location struct.
+    pub fn new(latitude: f64, longitude: f64) -> Self {
+        Self {
```

Comment:
just a thought: we could expect that both latitude and longitude are within a valid range

Reply:
Done